src/containers/AccountSettings/index.js

```diff
@@ -12,27 +12,32 @@ class Account extends Component {
       name: '',
       email: '',
       createdByGoogle: false,
+      createdByTwitter: false,
     },
     loading: true,
   };
 
-  componentDidMount() {
+  componentDidMount = async () => {
     const { firebase } = this.props;
     const userId = localStorage.getItem('userId');
-    firebase.db
+    const snapshot = await firebase.db
```

Comment:
You use async-await without using a try-catch!
- Handle errors here by adding a try-catch